.github/workflows/backport-queue.yml

```diff
@@ -0,0 +1,169 @@
+name: Backport Commit Queue
+
+on:
+  schedule:
+    # Run twice a week at 00:15 AM UTC on Sunday and Thursday.
+    - cron: 15 0 * * 0,4
```

Comment:
I would start with a manually run workflow to give it a try. Also if it keep it a manual step we can easily get around the two weeks problem by not

Reply:
If we have to run the workflow manually, then it has no value, I'd rather use the CLI

Reply:
I see a big value in not having to be at my workstation in order to kickstart a workflow, having the worfklow available in GitHub actions means I can just kickstart it anytime from my phone.

Reply:
Having it automated means you don't even need to pull up your phone though

Reply:
sure 👍 it makes total sense now that I understand the goal is not to land these commits on staging!

I'd still start with manual runs just to try it out but that's open to you ☺️